tests/backend/PortabilityService.Gateway.Tests/PortabilityService.Gateway.Tests.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\src\backend\PortabilityService.ConfigurationProvider\PortabilityService.ConfigurationProvider.csproj" />
```

Comment:
As mentioned before, I think the tests don't even need this.

Reply:
Removed this also